plugins/woocommerce-blocks/assets/js/blocks/product-filter/inner-blocks/rating-filter/style.scss

```diff
@@ -1,175 +1,8 @@
-@import "../../../shared/styles/style";
-
-.wc-block-rating-filter {
+.wp-block-woocommerce-product-filter-rating {
 	&.is-loading {
```

Comment:
Actually not convinced we need this but it does target the class on the empty loading state in the editor. I've left it as is for now, but happy to remove it because after removing it I didn't notice anything looking wrong in the styles.